profiler/src/main/java/com/splunk/opentelemetry/profiler/Configuration.java

```diff
@@ -51,6 +55,7 @@
     config.put(CONFIG_KEY_RECORDING_DURATION, DEFAULT_RECORDING_DURATION);
     config.put(CONFIG_KEY_KEEP_FILES, "false");
     config.put(CONFIG_KEY_MEMORY_ENABLED, String.valueOf(DEFAULT_MEMORY_ENABLED));
+    config.put(CONFIG_KEY_CALL_STACK_INTERVAL, DEFAULT_CALL_STACK_INTERVAL + " ms");
```

Comment:
Does the config parser tolerate spaces before unit?
```suggestion
    config.put(CONFIG_KEY_CALL_STACK_INTERVAL, DEFAULT_CALL_STACK_INTERVAL + "ms");
```

Reply:
I am 89% sure it does, but it's probably cleaner without the space anyway.